components/sections/Footer.tsx

```diff
@@ -55,20 +55,17 @@ export default function Footer(): ReactElement {
                 _hover={{}}
                 fontSize={["xs", "xs", "xs", "sm"]}
                 href="https://github.com/cusec/2021"
-                rel="noopener noreferrer"
-                target="_blank"
+                isExternal
               >
                 Open Source
               </FooterLink>
               <FooterLink
```

Comment:
Missing href or TBD?

Reply:
TBD :sweat_smile: I'm thinking it'd be nicer to have all the attributions listed in the modal component you made!

Reply:
That sounds good!